gems/pending/spec/miq_vm/miq_azure_vm_image_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative './miq_vm_instance_methods_shared_examples'
+require 'ostruct'
+require 'azure-armrest'
+require 'MiqVm/miq_azure_vm'
+require 'vcr'
+
+describe MiqAzureVm do
+  before(:all) do
```

Comment:
You really should never do `before(:all)` in tests...what exactly is the TestEnvHelper for...we don't do anything like that in the vmdb specs, which is why I ask.